operations/customer-success/README.md

```diff
@@ -129,3 +132,31 @@ Below are detailed steps on how to log a Customer Reference in Salesforce.
    - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that the mattermost cannot use their logo or publish case studies.
    - **Notes:** Field to add any additional details.
 4. Select **Save**.
+
+### Customer Risk
+
+Risk is the chance or probability that a customer will non-renew their existing Enterprise license. The Customer Success team categorizes risk into two buckets:
```

Comment:
I understand "non-renew" is the term we use, but wondering if we could say "not renew" in this case? 0/5.

Reply:
@justinegeffen I'm good with that change.